monero-wallet-gui.pro

```diff
@@ -91,7 +91,8 @@ LIBS += -L$$WALLET_ROOT/lib \
         -lwallet_merged \
         -lepee \
         -lunbound \
-        -leasylogging
+        -leasylogging \
+        -lreadline \
```

Comment:
Thanks  for the fix. I suggest escape slash can be left out since we are at the end.

Reply:
Ending slash is good because that way entries can be added to the list without changing other lines. This is good for not polluting git blame. The current diff is an example of bad diff that is due to the lack of the slash.